cupyx/linalg/linalg.py

```diff
@@ -0,0 +1,99 @@
+import numpy
+
+import cupy
+from cupy import cuda
+from cupy.cuda import cublas
+from cupy.cuda import device
+from cupy.linalg import util
+
+if cuda.cusolver_enabled:
+    from cupy.cuda import cusolver
+
+
+def invh(a):
+    """Compute the inverse of a Hermitian matrix.
+
+    This function computes a inverse of a real symmetric or complex hermitian
+    matrix.
```

Comment:
Can we document the behavior for cases where `a` is not Hermitian?

Reply:
Sure, I updated docs.